### ISSUE TITLE
Make a “PlanningAlerts Backers” page

### ISSUE THREAD

**Issue by equivalentideas** [openaustralia/planningalerts#1056]
A page that:

1. explains the concept
2. celebrates the Crew
3. helps people join

These are things that we think are important for people who donate to the project:

* By joining the crew you are making PlanningAlerts possible, which is an amazing thing. It let us send out 100,000 alert emails, like the ones you get, to people this week for example.
* All these other great people like you are supporting PlanningAlerts by joining the Crew.
* You feel appreciated by the OpenAustralia Foundation, who’s values you share.
* Henare, Luke, Kat, and the OpenAustralia Foundation team are speaking to you *personally*.

This page should have a recognisable colour that is consistent through the site and email alerts to signify that this is a special thing.

**Comment by equivalentideas** [comment]
I'm looking at some prior art to see how people are presenting this kind of thing.

## Canadaland podcast

Here’s an example from CanadaLand, a podcast that has been extremely successful in getting listeners to support it over a few years.

![screen shot 2016-11-25 at 1 55 23 pm](https://cloud.githubusercontent.com/assets/1239550/20614777/eec99ab6-b319-11e6-85f6-06cecb496272.png)

I think these things work well about this:

* It makes a very direct ask from the thing people love, the podcast, to them the listener: “CanadaLand exists because people like you support it”, “support us”.
* people are being asked to support the thing Canadaland makes: Independent *Canadian* Journalism. They're putting forward the thing that's really unique to them, that they are one of the few local independent outlets.
* you can see what your money is going to, so you really feel like you're directly enabling that work—it will not happen without you.
* you’re invited to see exactly how much money they get through this, which feels very transparent—a value that's important to their listeners and that they expect Canadaland to demand of others. The goal thing makes the money bit make sense.

I'd love to know how many people fund this here, but I like that they've kept it as simple as possible.

I think this does a really effective job of achieving our three goals for this page above in simple way.

In their actual show they always directly say that the show is ‘only possibly with all their great supporters‘. They also start every show with two people who fund it answering the question “why do you think Canadaland is awesome?”. It's always about the important of independent, public interest, local journalism. This makes me want to go have a good look at [what our donors said when we asked them a similar question](https://docs.google.com/spreadsheets/d/1QcL9pKFJ5GMQ_RWzHqQdaKq89sfebB9OgbamlLimd_E/edit#gid=2094408790).

At the bottom of each podcast page they have a variation on the box above:

![screen shot 2016-11-25 at 2 32 37 pm](https://cloud.githubusercontent.com/assets/1239550/20614956/4a188592-b31c-11e6-98bd-1c6065d516b5.png)

**Comment by equivalentideas** [comment]
## [The Guardian](https://membership.theguardian.com/au/supporter)

![screen shot 2016-11-25 at 12 14 02 pm](https://cloud.githubusercontent.com/assets/1239550/20615244/43be3608-b31f-11e6-898a-31abc8834543.png)

I don't think this example makes it seem like members make the Guardian possible, which is true they’re largely funded by other means. It does celebrate their members through “Supporters keep our journalism fearless and free to tell the stories that matter”. They have a big forces on the benefits, and the member program is much more like a club where you get something than the idea of “backers” or “patrons”.

The big image is interesting. It's like they're saying, if you're the type of person at this protest, then become a member which is a similar action. Though then the benefits focus kinda undercuts it.

I find this page too complex, there's too many different messages. I feel more like I'm being sold something than with the Canadaland example.

**Comment by equivalentideas** [comment]
## [GetUp!](https://www.getup.org.au/)

![screen shot 2016-11-25 at 12 24 42 pm](https://cloud.githubusercontent.com/assets/1239550/20615392/0ded3b44-b321-11e6-90f7-b1f6b779a2f3.png)

Getup are really positioning the organisation as a ‘movement’ of their members “Join the movement of 1,076,149 Australians”. It's cool to see the total number of people on their email list, it feels massive—but knowing that this is just people signed up to a mail list makes me feel less good about it. They're also pushing their members forward this the stream of actions taken by people.

On their donation page this is much more about the work they do, and impact they have, that people “chip in” to support. “We're moving Australia towards a more fair, flourishing, and just tomorrow. But we can't get there without you. When you support GetUp, you're not just funding political noise. You're powering a million-strong, strategically savvy organisation that makes real change happen. ” “CAN YOU CHIP IN?” plays down the weight of the action of donating, but also makes it seem like your responsibility—that tone doesn't feel right to for us. This page also feels very complex, even more complex than the Guardian example. I feel like we're striving for simpler.

![screen shot 2016-11-25 at 3 13 44 pm](https://cloud.githubusercontent.com/assets/1239550/20615461/d576d328-b321-11e6-9547-1b90a973708d.png)

**Comment by equivalentideas** [comment]
## [ProPublica](https://www.propublica.org/donate/)

This has an epic form, and the messaging is that donations help, but you get the sense that they aren't their main funding source. However, something I really like about this, that they under-play, is the big photo of one of their reporters on the job. Unfortunately the caption is tiny and at the bottom, so I didn't even understand what this was for a while. I really like the sense that you can see a person doing the work you’re contributing for them to do.

I really like the caption text as well. It's very personal, lining up with our idea to name the people you're helping: “ProPublica reporter Marshall Allen in a Detroit hospital reporting for Surgeon Scorecard.” It's specific and feels very real.

![screen shot 2016-11-28 at 3 24 43 pm](https://cloud.githubusercontent.com/assets/1239550/20657663/28dbe7a6-b57f-11e6-9d72-aca03b86c0e3.png)

**Comment by equivalentideas** [comment]
Here’s an idea for how our page could work. Here's the bits and see the sketch below.

1. Leading with one big statement that, like the CanadaLand example, captures the thing you’re supporting, something like “You should be informed and have your say on changes to your local area”.
2. The bit you interact with, starting with a short sentence that is about the backer concept, e.g. “PlanningAlerts exists because 37 people like you back it with a small monthly donation”. Then the amount setter (defaulting to something like $3.50), and a big call-to-action button, e.g. “back PlanningAlerts”, “become a backer”, or “Join the PlanningAlerts Backers”. The button activates the Stripe card form.
3. After a bit of space, to put main focus on the payment section, a bit to emphasis the amazingness of PlanningAlerts enabled by these people. I was thinking the big vanity stats, but just 3, [like on morph.io](https://morph.io/).
4. Emphasis the personal connection, and the people your money goes to support, at the end of the page a big thank you from the humans at the OpenAustralia Foundation, naming us. It would also be great to show a bunch of the awesome volunteer contributors at the hackfest ([Dave from Bike Sydney got a nice black and white photo at our May Hackfest at Frontyard](https://twitter.com/BIKESydney/status/723686583786852353).

I'm thinking some bold colours behind the sections. Possibly a darker background behind the stats bit.

Together these bits add up to a lot of things to get excited about and a simple action they can take. What do you think @henare ? My gut is that this could work well, or we could go simpler and cut at the button, and then save the other stuff for the thankyou page.

I also just had an idea that we could have a google map with pins of alerts faded back behind the top section, reminding these passionate supporters of the thing that they love and get in their emails.

![img_4929](https://cloud.githubusercontent.com/assets/1239550/20658151/555c05a6-b582-11e6-9fcb-7cadcc36a2a2.JPG)

**Comment by equivalentideas** [comment]
I've been playing with the main front section in HTML. Initially I went for a layout like in the sketch, but that wasn't working so well. The next iteration has been a two column layout with the main text message on the left and the simpler form on the right. I think this is working way better as it keeps the form simpler and quicker to get to. Also the text seems more striking as well. With @henare’s feedback I've also made the form button better describe exactly what's happening rather than use the 'backers' concept.

I've had a little play with some coloured backgrounds, but it's not grabbing me yet. I think the background needs some texture or image to make it feel less shiny and more human.

## One column

![screen shot 2016-11-30 at 3 01 01 pm](https://cloud.githubusercontent.com/assets/1239550/20783177/ae771186-b7d9-11e6-95f9-b7a8351a774e.png)

## Two column

![screen shot 2016-12-01 at 3 10 26 pm](https://cloud.githubusercontent.com/assets/1239550/20783185/b43b2918-b7d9-11e6-8e06-d126d53bb069.png)

## Colour tests

![screen shot 2016-12-01 at 3 35 46 pm](https://cloud.githubusercontent.com/assets/1239550/20783601/68cde800-b7dc-11e6-9948-cb4306f079a8.png)
![screen shot 2016-12-01 at 3 35 24 pm](https://cloud.githubusercontent.com/assets/1239550/20783602/68d1c970-b7dc-11e6-879f-ce9b47f55743.png)
![screen shot 2016-12-01 at 3 33 08 pm](https://cloud.githubusercontent.com/assets/1239550/20783603/68d6f4b8-b7dc-11e6-8a44-0d5e8cc8ec3b.png)
![screen shot 2016-12-01 at 3 32 51 pm](https://cloud.githubusercontent.com/assets/1239550/20783604/68dbd492-b7dc-11e6-90b6-5583325b4d9a.png)
![screen shot 2016-12-01 at 3 32 35 pm](https://cloud.githubusercontent.com/assets/1239550/20783605/68e178de-b7dc-11e6-81fa-f27953902597.png)
![screen shot 2016-12-01 at 3 23 19 pm](https://cloud.githubusercontent.com/assets/1239550/20783606/68e41d96-b7dc-11e6-934d-c656e4c14393.png)

**Comment by equivalentideas** [comment]
I'm going to sit with that for a while and see if implementing some [bits for down the page](https://github.com/openaustralia/planningalerts/issues/1056#issuecomment-263194163) help develop it.

**Comment by equivalentideas** [comment]
I've worked down the page and brought in the elements I was planning from the sketch. After a bit of iteration I think this is working pretty well.

I've brought in a placeholder image at the top, but the idea is to show a human actually being informed through the PlanningAlerts emails they know and love.

I got great feedback from Miška Mandic to make the “Thanks” section a quote from the team. I think this works well because it makes it way more personal to be in our voices, rather than the website’s voice.

I'll have to reshoot the top image, and I'm trying to get a higher quality version of the hackfest photo.

I also wanna give the numbers the same roll animation as morph.io to bring them to life a bit.

![2016-12-02_mockup-fullpage](https://cloud.githubusercontent.com/assets/1239550/20826011/8d50dc18-b8af-11e6-8731-354a37c4fc73.png)

Next steps are to do the thanks page and the make this actually work.
